### PR TITLE
build(design): Bump postcss in design package

### PR DESCRIPTION
**Pull request by MichaelParadis**
## Motivations

Necessary package bumps for Node 18 upgrade

### Added

- `@csstools/postcss-extract` at `v2.0.1`

### Changed

 
- `postcss-calc` from `7.0.2` => `8.2.3`
- `postcss-cli` from `6.1.2` => `10.1.0`
- `postcss-custom-properties` from `8.0.10` => `13.1.4`
- `postcss-import` from `12.0.1` => `15.1.0`

### Removed

 
- `postcss-custom-properties` announced they were no longer maintaining `exportTo` & `importFrom`, hence why `postcss-extract` was added. 
Context: https://github.com/csstools/postcss-plugins/discussions/192#discussioncomment-4264336

## Testing

- extensive testing has been done in JO, mobile (iOS and android) & both Storybook's
---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

**Comment by rodrigoeidelvein** [comment]
@MichaelParadis Do you already have a prerelease for this branch?

**Review by rodrigoeidelvein — CHANGES_REQUESTED** [review]
I noticed a difference in the typography tokens when running `npm run build` inside `packages/design` and also confirmed that it was different when I installed the prerelease in JO.

Left is the branch, right is master
- `foundation.js`
image

- `foundation.scss`
image

weird that `extravagant` is 16 in both branches 👀

**Comment by MichaelParadis** [comment]
New pre-release
```
 - @jobber/components@4.4.3-pre.9+02e01eb4
 - @jobber/design@0.31.2-pre.9+02e01eb4
```

**Comment by taylorvnoj** [comment]
- [x] iOS looks good with latest pre-release

File: packages/design/buildFoundation.js [hunk 1]
```diff
@@ -2,13 +2,33 @@
...
-    exportTo: ["src/foundation.js"],
+  postcssExtract({
+    queries: {
+      customProperties: 'rule[selector*=":root" i] > decl[value]',
+    },
+    results: yourResults => {
+      const mappedResults = yourResults.customProperties.reduce(
+        (acc, { prop, value }) => {
+          if (!(prop in acc)) {
+            acc[prop] = value; //.trim(); //.replaceAll("\n", "");
```
Comment (rodrigoeidelvein) [inline packages/design/buildFoundation.js:18]: Ohhh, that was clever. I think we can only remove the comment! Good solution to not rewrite the values!

**Review by rodrigoeidelvein — CHANGES_REQUESTED** [review]
It's looking good! And even fixed the `extravagant` and `smaller` values
image

The only thing is that we can remove that comment, then we're good to go

File: packages/design/buildFoundation.js (hunk 1 above)
Comment (MichaelParadis) [reply packages/design/buildFoundation.js:18]: Ahh, thanks. I missed that

Comment (MichaelParadis) [reply packages/design/buildFoundation.js]: @rodrigoeidelvein Updated it

**Review by rodrigoeidelvein — APPROVED** [review]
LGTM!

**Pull request merged**